draft-ietf-scitt-architecture.md

```diff
@@ -363,36 +366,41 @@ This section describes at a high level, the three main roles and associated proc
 
 ## Signed Statement Issuance and Registration
 
-### Issuer Identity
+### Structure and Content of Signed Statements
+
+Before an Issuer is able to produce Signed Statements, it must first create an identifier and obtain an identity document that is acceptable to the Transparency Service.
```

Comment:
Do we want to make registering identity documents mandatory prior to accepting signed statements from issuers?

This makes the replay verification simple, find the public key in the log, then find each signature.

Reply:
I would very, very much like that. They'll be used in Registration Policies and those also should be registered. So - yeah +1 (to automation, resilience, and transparency!)